src/amo/reducers/addonsByAuthors.js

```diff
@@ -172,6 +174,8 @@ const reducer = (
   switch (action.type) {
     case FETCH_ADDONS_BY_AUTHORS: {
       const newState = deepcopy(state);
+      const { addonType: type } = action.payload;
+      const addonType = getAddonTypeFilter(type);
```

Comment:
mmh another question: do we actually have to modify this reducer?

Reply:
you're right..I thought we needed this but we don't :/